### ISSUE TITLE
Some pages have same display

### ISSUE THREAD

**Issue by TheWinchesters** [Sarjuuk/aowow#155]
I do not know why but some pages like /?help=screenshots-tips-tricks show content from /?help=commenting-and-you.
This is the db table:
https://ibb.co/er62S0

What did I do wrong?

I can edit it by hand, but where can I find the content for those pages?

**Comment by TheWinchesters** [comment]
Found the content I need to change, I will change it by hand.
Why did this happen? No idea.

**Comment by TheWinchesters** [comment]
Finally, if someone has the same problem, just delete the entries and insert again.
The insert query needed is in db_structure.sql

**Comment by Sarjuuk** [comment]
Why did this happen? No idea.

Pretty much this. Either you are the first with this issue or nobody else bothered reporting it.
Every other table with predefined content is fine?

**Comment by TheWinchesters** [comment]
Everything else is fine.
The problem is only with this table, in those entries where type and typeId are NULL.

I can tell you how to reproduce the problem, (or it is still just a problem for me).
I speak spanish so I was trying to translate those articles.
- I made the first one (tooltips), making a copy of the original one, changing locale 0 to 6 and spanish content.
- Everything good.
- I made the second one (aboutus).
- Now, tooltips and aboutus have the same text in 'article'.
